### PR TITLE
[hi] Localize horizontal-scaling.md to Hindi

### LINKED ISSUE #1623: Localise Horizontal Scaling page in hindi
The Hindi version of Horizontal Scaling is not available.
Link: https://glossary.cncf.io/horizontal-scaling/

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
Signed-off-by: ANUSHKA SAXENA <anushkasaxena102@gmail.com>

### Describe your changes
Created a file `horizontal-scaling.md` to localize the page: https://glossary.cncf.io/horizontal-scaling/ into Hindi 

### Related issue number 
Resolves #1623 

### Checklist before opening this PR 
- [x] This PR does not contain plagiarism
- [x] I have signed off on all commits

### CODE REVIEW

File: content/hi/horizontal-scaling.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: क्षैतिज स्केलिंग (Horizontal Scaling)
+status: Feedback Appreciated
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:3]:
```suggestion
status: completed
```

File: content/hi/horizontal-scaling.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+category: संकल्पना
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:4]:
```suggestion
category: concept
```

File: content/hi/horizontal-scaling.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+tags:  ["Infrastructure"]
+---
+
+## यह क्या है
+
+क्षैतिज स्केलिंग एक ऐसी तकनीक है जिससे एक सिस्टम की क्षमता को अधिक [नोड्स](/nodes/) जोड़कर बढ़ाया जाता है, बजाय अलग-अलग नोड्स में अधिक कंप्यूट संसाधन जोड़ने के (इसे [लम्बवत स्केलिंग](/vertical-scaling/) के रूप में जाना जाता है)। मान लीजिए, हमारे पास 4GB RAM की एक प्रणाली है और इसकी क्षमता को 16GB RAM तक बढ़ाना चाहते हैं, इसे क्षैतिज रूप से स्केल करने का मतलब 16GB RAM सिस्टम पर जाने के बजाय 4 x 4GB RAM जोड़ना है।
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:10]:
```suggestion
क्षैतिज स्केलिंग एक ऐसी तकनीक है जिससे एक सिस्टम की क्षमता को अधिक [नोड्स](/nodes/) जोड़कर बढ़ाया जाता है, 
बजाय अलग-अलग नोड्स में अधिक कंप्यूट संसाधन जोडे, जिसे [लम्बवत स्केलिंग](/vertical-scaling/) के रूप में जाना जाता है। 
मान लीजिए, हमारे पास 4GB RAM की एक प्रणाली है और हम इसकी क्षमता को 16GB RAM तक बढ़ाना चाहते हैं, 
इसे क्षैतिज रूप से स्केल करने का मतलब 16GB RAM सिस्टम पर जाने के बजाय 4 x 4GB RAM जोड़ना है।
```

File: content/hi/horizontal-scaling.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+यह दृष्टिकोण कार्यभार को बेहतर ढंग से वितरित करने के लिए नए इंस्टैंस, या [नोड्स](/nodes/) जोड़कर किसी एप्लिकेशन की कार्यक्षमता को बढ़ाता है। सरल शब्दों में, इसका उद्देश्य व्यक्तिगत सर्वर की क्षमता बढ़ाने के बजाय सर्वर पर आने वाले भार को कम करना है।
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:12]:
```suggestion
यह तरीका कार्यभार को बेहतर ढंग से वितरित करने के लिए नए इंस्टैंस, या [नोड्स](/nodes/) जोड़कर किसी एप्लिकेशन की कार्यक्षमता को बढ़ाता है। 
सरल शब्दों में, इसका उद्देश्य व्यक्तिगत सर्वर की क्षमता बढ़ाने के बजाय सर्वर पर आने वाले भार को कम करना है।
```

File: content/hi/horizontal-scaling.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## समस्या
+
+जैसे ही किसी एप्लिकेशन की मांग उस एप्लिकेशन इंस्टेंस की वर्तमान क्षमता से ज़्यादा बढ़ती है, हमें सिस्टम को [स्केल](/scalability/) करने (क्षमता जोड़ने) का एक तरीक़ा खोजने की आवश्यकता होती है। हम या तो सिस्टम में अधिक नोड्स (क्षैतिज स्केलिंग) जोड़ सकते हैं या मौजूदा नोड्स (लंबवत स्केलिंग) में अधिक कंप्यूट संसाधन जोड़ सकते हैं।
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:16]:
```suggestion
जैसे ही किसी एप्लिकेशन की मांग उस एप्लिकेशन इंस्टेंस की वर्तमान क्षमता से ज़्यादा बढ़ती है, 
हमें सिस्टम को [स्केल](/scalability/) करने के लिए एक तरीक़ा खोजने की आवश्यकता होती है। हम या तो सिस्टम में 
अधिक नोड्स (क्षैतिज स्केलिंग) जोड़ सकते हैं या मौजूदा नोड्स (लंबवत स्केलिंग) में अधिक कंप्यूट संसाधन जोड़ सकते हैं।
```

File: content/hi/horizontal-scaling.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+## समाधान
+
+क्षैतिज स्केलिंग एप्लिकेशन्स को अंतर्निहित क्लस्टर द्वारा प्रदान की जाने वाली किसी भी सीमा तक स्केल करने की अनुमति देता है। सिस्टम में अधिक इन्सटेंसेस जोड़कर, ऐप अधिक संख्या में अनुरोधों को संसाधित कर सकता है। यदि एक एकल नोड प्रति सेकंड 1,000 अनुरोधों को संभाल सकता है, तो प्रत्येक अतिरिक्त नोड प्रति सेकंड लगभग 1,000 अनुरोधों की कुल संख्या में वृद्धि कर सकेगा।  यह एप्लिकेशन को विशेष रूप से किसी भी नोड की क्षमता बढ़ाने की आवश्यकता के बिना समवर्ती रूप से अधिक कार्य करने के लिए सक्षम बना देता है।
```
Comment (aj11anuj) [inline content/hi/horizontal-scaling.md:20]:
```suggestion
क्षैतिज स्केलिंग एप्लिकेशन्स को अंतर्निहित क्लस्टर द्वारा प्रदान की जाने वाली किसी भी सीमा तक स्केल करने की अनुमति देता है। 
सिस्टम में अधिक इन्सटेंसेस जोड़कर, ऐप अधिक संख्या में अनुरोधों को संसाधित कर सकता है। 
यदि एक एकल नोड प्रति सेकंड 1,000 अनुरोधों को संभाल सकता है, तो प्रत्येक अतिरिक्त नोड प्रति सेकंड 
लगभग 1,000 अनुरोधों की कुल संख्या में वृद्धि कर सकेगा।  यह एप्लिकेशन को विशेष रूप से किसी भी नोड की क्षमता बढ़ाने की 
आवश्यकता के बिना समवर्ती रूप से अधिक कार्य करने के लिए सक्षम बना देता है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]

**Review by kumarankit999 — APPROVED** [review]
looks good to me, What others Approvers say?

**Comment by aj11anuj** [comment]
@SaxenaAnushka102 your PR is ready to get merged, but you will have to change the target branch from 'cncf: main' to 'cncf: dev-hi' branch

**Comment by SaxenaAnushka102** [comment]
I've changed the branch to `dev-hi`, is it fine now @aj11anuj ?
Please let me know, thanks!

**Comment by bishal7679** [comment]
Thank you @SaxenaAnushka102
Its ready to merge!

**Pull request merged**